components/dashboard/src/onboarding/job-roles.ts

```diff
@@ -9,13 +9,13 @@ export const JOB_ROLE_OTHER = "other";
 export const getJobRoleOptions = () => {
     return [
         { value: "", label: "Please select one" },
-        { value: "software-eng", label: "Software Engineer" },
+        { value: "software-eng", label: "Software Engineering" },
         { value: "data", label: "Data / Analytics" },
-        { value: "academics", label: "Academic (Student, Researcher)" },
-        { value: "enabling", label: "Enabling team (Platform, Developer Experience)" },
-        { value: "team-lead", label: "Team / Function Lead" },
+        { value: "academics", label: "Academia (Student, Researcher)" },
+        { value: "enabling", label: "Platform or Developer Experience" },
+        { value: "team-lead", label: "A Team or Function Lead role" },
```

Comment:
**Suggestion(non-blocking):** Could we add "lead" after team lead? I noticed some people submitted "team lead" in other, but did not notice here, my theory is because we say "team / function lead" so on scanning "team lead" does not appear. 

```suggestion
        { value: "team-lead", label: "A Team Lead or Function Lead role" },
```